File: app/src/main/java/me/blog/korn123/easydiary/helper/EasyDiaryApplication.java
package me.blog.korn123.easydiary.helper;

import android.app.Application;

import io.realm.Realm;

/**
 * Created by CHO HANJOONG on 2017-03-16.
 */

public class EasyDiaryApplication extends Application {
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
    }
}

File: app/src/main/java/me/blog/korn123/easydiary/helper/EasyDiaryApplication.kt
package me.blog.korn123.easydiary.helper

import android.content.Context
import androidx.multidex.MultiDexApplication
import dagger.hilt.android.HiltAndroidApp
import io.realm.Realm

/**
 * Created by CHO HANJOONG on 2017-03-16.
 */

@HiltAndroidApp
class EasyDiaryApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        context = this
    }

    companion object {
        var context: Context? = null
    }
}